src/components/Summary/NoFeeSummary.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <v-card flat id="no-fee-summary" v-if="isSummaryMode">
+    <div class="summary-header">
+      <v-layout row class="mx-0">
+        <v-flex xs9>
+          <v-icon class="my-n1" color="#38598A">mdi-handshake</v-icon>
```

Comment:
Future: we need to think about the Vuetify theme colours so we can use a name here instead of hex code.

Reply:
ahh right, will update this now!